config/locales/server.en.yml

```diff
@@ -2587,6 +2587,8 @@ en:
     actions:
       grant_admin:
         description: "For additional security measures, you need to confirm your 2FA before %{username} is granted admin access."
+      discourse_connect_provider:
+        description: "%{hostname} has requested that you confirm your 2FA. You'll be redirected to %{sso_return_url} once you confirm your 2FA."
```

Comment:
Minor, but I think just the domain would be better here.

Reply:
Can you explain why? If the user will be redirected to e.g. `meta.discourse.org`, shouldn't it say `meta.discourse.org` (the hostname) rather than just `discourse.org` (domain)?

Reply:
`meta.discourse.org` is good. `https://meta.discourse.org/random-sso-return-path` is less good.

Reply:
Sorry, it's the `sso_return_url`, I have an objection to.

Reply:
Ahh I see! Yeah sure, I will remove it 👍